apps/src/code-studio/components/stageExtras/StageExtras.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import msg from '@cdo/locale';
 import BonusLevels from './BonusLevels';
-import CreateSomething from './CreateSomething';
+import ProjectWidget from '../../../templates/projects/ProjectWidget.jsx';
```

Comment:
also, you dont need to specify `.jsx` in the path.

There's been some debate in general about relative vs. absolute path. Brad has complained that his editor doesnt handle the absolute path as well. I much prefer them, in part because it makes copy/pasting paths between files easier.